libs/components/forms/src/lib/modules/radio/fixtures/radio-group-reactive.component.fixture.ts

```diff
@@ -25,27 +27,31 @@ export class SkyRadioGroupReactiveFixtureComponent implements OnInit {
     { name: 'Harry Mckenzie', disabled: false },
   ];
 
-  public radioControl: UntypedFormControl;
-  public radioForm: UntypedFormGroup;
+  public radioControl!: UntypedFormControl;
+  public radioForm!: UntypedFormGroup;
 
   public required = false;
 
   public showRadioGroup = true;
 
-  public tabIndex: number;
+  public tabIndex: number | undefined;
 
   @ViewChild(SkyRadioGroupComponent)
-  public radioGroupComponent: SkyRadioGroupComponent;
+  public radioGroupComponent: SkyRadioGroupComponent | undefined;
+
+  #fb: UntypedFormBuilder;
 
-  constructor(private fb: UntypedFormBuilder) {}
+  constructor(fb: UntypedFormBuilder) {
+    this.#fb = fb;
```

Comment:
Let's make this variable consistent with other implementations, `formBuilder: UntypedFormBuilder` and `#formBuilder`.

Reply:
Done